Library/Homebrew/test/testing_env.rb

```diff
@@ -20,6 +20,25 @@
   abort "Run `bundle install` or install the mocha and minitest gems before running the tests"
 end
 
+if ENV["HOMEBREW_TESTS_COVERAGE"]
+  # Ensure all Ruby files are `require`d for an accurate coverage measurement
+  Dir[File.expand_path("../../**/*.rb", __FILE__)].each do |f|
+    excluded_core_files = [
+      # scripts
+      "build.rb", "postinstall.rb", "test.rb",
+      # mocked files
+      "config.rb", "global.rb",
+    ]
+
+    next if excluded_core_files.any? { |file| f.end_with?("Homebrew/#{file}") }
+
+    # ignore test files
+    next if f.include?("/test/")
+
+    require f
+  end
+end
```

Comment:
I would suggest don't include this even for coverage. Because, I think to get correct test result is more important than coverage. 

Also it's worthy nothing to mention, even something like this, the coverage percent is still wrong. We don't count coverage of codes used when running build or postinstall script. 

As for coverall, I think it's fine to keep it. Because I know the percent is wrong, at the same time, it does a good job to tracking which lines are already covered(notice to that it doesn't track what lines not covered)